bin/docker-dev-setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+echo '[info]: Configuring container for development'
+apk add make
+echo '[info]: Installing development dependencies'
+cp package.json -f ../
+cd ../
+npm install --also=dev && cd ./app
```

Comment:
What does the `--also` flag do?

Reply:
It installs the dev dependencies as that's all that is missing.

Reply:
Are you perhaps meaning to use the `--only` flag? I can't find any documentation on `--also`

Reply:
Yeah maybe `--only` is what I need. I came across `--also=dev` because I used `npm install --dev`, to which npm replied: `npm WARN install Usage of the `--dev` option is deprecated. Use `--also=dev` instead.`